src/cmd/flux-ping.c

```diff
@@ -262,6 +262,7 @@ void parse_service (const char *s, uint32_t *nodeid, char **topic)
 int main (int argc, char *argv[])
 {
     int pad_bytes;
+    char *cmdusage = "[OPTIONS] <target>";
```

Comment:
Very minor, but none of our other commands (after spot checking) use the `<arg>` style like this, and even the `flux ping` man page uses `[OPTIONS] target`. Most of our commands have adopted `[OPTIONS]  TARGET` style, but I'd be fine with just matching the man page.

I'd actually be fine merging as-is, but thought I'd mention it for consistency's sake (and you know some  code janitor is going to come along in the future and change it, so might as well do it now :stuck_out_tongue_winking_eye: )

Reply:
ahh good catch / idea, will adjust.